tests/Checks.hs

```diff
@@ -2,6 +2,7 @@
 module Main where
```

Comment:
it's just the main module of the 2nd work-in-progress test-suite. I'll add the `(main)` but it make no practical difference I think